filter.rs.handlebars

```diff
@@ -136,6 +159,14 @@ impl Filter {
                 // In the non-simulator version, we will send the result to storage.  Given this is
                 // a simulation, we will write it to a file.
                 {{#each rust_blocks}}{{{this}}}{{/each}}
+
+                {{#with result}}
+                {{#if this.Return}}{{#with this.Return}}
+                fs::write("result.txt", {{this.id}}).expect("Unable to write file");
```

Comment:
Ideally, the simulator should have storage node that writes to a file at the end or provides a testing API. Not sure if this is the right way to go. This should be a RPC request. Maybe make this a TODO

Reply:
Good call